clusters/base/falco-sci.yaml

```diff
@@ -167,7 +166,7 @@ data:
               "showLineNumbers": false,
               "showMiniMap": false
             },
-            "content": "\n\n\n# [SCI Equation = ](https://sci-guide.greensoftware.foundation/)\n# [(E * I) + M per R](https://sci-guide.greensoftware.foundation/)",
+            "content": "\n\n\n# [SCI Equation = ](https://sci-guide.greensoftware.foundation/)\n# [(E * I * PUE) + M per R](https://sci-guide.greensoftware.foundation/)",
```

Comment:
I would revert this to the top-level SCI formula `(E * I)`

Reply:
I was also hesitating on this, and at the end I went for it to avoid having multiple formulas on different places, also because people become aware of the components that contribute to the emissions calculation. I think that this also relates to [this comment](https://github.com/cncf-tags/green-reviews-tooling/pull/76/files/3a6266ceae99f40aaa367174ffb899385caf1d50#r1529137645)